datafusion/sql/src/relation/mod.rs

```diff
@@ -33,7 +33,10 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         let (plan, alias) = match relation {
             TableFactor::Table { name, alias, .. } => {
                 // normalize name and alias
-                let table_ref = object_name_to_table_reference(name)?;
+                let table_ref = object_name_to_table_reference(
+                    name,
+                    self.options.enable_ident_normalization,
+                )?;
```

Comment:
This code could potentially be improved by making a `self.object_name_to_table_reference` rather than having to pass ` self.options.enable_ident_normalization,` at each callsite, but we can do that as a follow on PR

Reply:
Make sense, I can fix this in following PR.